app/routes.py

```diff
@@ -1,2 +1,291 @@
-from flask import Blueprint
+# from flask import Blueprint
+from app import db
+from flask import request, Blueprint, make_response
+from flask import jsonify
+import datetime
+from datetime import datetime 
+import requests
+from flask import current_app as app
+import os 
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+# from datetime import timedelta
+
+
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def customers():
+    
+    if request.method == "GET":  
+        customers = Customer.query.all()
+        customers_response = []
+
+        for customer in customers:
+        
+            customers_response.append(customer.to_json())
+            
+        return jsonify(customers_response)
+    
+
+@customers_bp.route("", methods=["POST"], strict_slashes=False)
+def create_customers():
+    
+    
+    request_body = request.get_json()
+    if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
```

Comment:
It would make a good idea to create a helper function to verify that the request body has the correct fields.